### PR TITLE
chore: update UI addNFTCollection

### PR DESCRIPTION
**Pull request by NMINHDAI**
**What does this PR do?**

-   [x] My bugfix

**How to test**

-   Create a detailed description of what you need to do to set this PR up. ie: Does it need migrations? Do you need to install something?
-   Create a step by step list of what the engineer needs to do to test.

**Edge cases**

-   What are the edge cases that you might see in this PR?

**Media (Loom or gif)**

-   Insert media here (if applicable)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,9 +11,9 @@
     "@types/cron": "^2.0.0",
     "@types/jest": "^27.0.2",
     "@types/koa": "^2.13.1",
-    "@types/koa__cors": "^3.0.2",
     "@types/koa-bodyparser": "^4.3.3",
     "@types/koa-router": "^7.4.2",
+    "@types/koa__cors": "^3.0.2",
```
Comment (trkhoi) [inline package.json:16]: why need change this ?

File: src/commands/community/nft/add.ts [hunk 2]
```diff
@@ -35,6 +35,12 @@ async function executeNftAddCommand(args: string[], msg: Message) {
...
   const dataChain = await respChain.json()
   switch (respCollection.status) {
+    case 400:
+      return buildDiscordMessage(
+        msg,
+        "NFT",
+        "Already added. Nft is in sync progress"
+      )
     case 200:
       return buildDiscordMessage(msg, "NFT", "Successfully add new collection")
```
Comment (trkhoi) [inline src/commands/community/nft/add.ts:45]: return buildDiscordMessage(msg, "NFT", "Successfully add new collection to queue")

**Review by trkhoi — CHANGES_REQUESTED** [review]

**Review by trkhoi — APPROVED** [review]

File: src/commands/community/nft/add.ts [hunk 3]
```diff
@@ -35,8 +35,18 @@ async function executeNftAddCommand(args: string[], msg: Message) {
   const errorMessageCollection = dataCollection.error
   const dataChain = await respChain.json()
   switch (respCollection.status) {
+    case 400:
+      return buildDiscordMessage(
```
Comment (trkhoi) [inline src/commands/community/nft/add.ts:39]: add this to default case and catch exactly error

**Review by trkhoi — CHANGES_REQUESTED** [review]
